### PR TITLE
Opening the mini-map reverts the size of the land tool to 1x1, regardless of what was selected before. #7382

### LINKED ISSUE #7382: Opening the mini-map reverts the size of the land tool to 1x1, regardless of what was selected before.
**OS:** Windows 8
**Version:** e.g. 0.1.3
**Commit/Build:** 69bb1cd

Opening the mini-map reverts the size of the land tool to 1 square, regardless of how large an area you had set the tool to cover initially. 
This is kind of a bizarre behavior between two things I wouldn't expect to be related at all, but if you're using the land tool and you've increased the size of its effect (say to a 3x3 square), and then you open up the map, the land tool's size will go back to 1x1 all on its own. This happens both when using the tab key shortcut for opening the map _and_ when opening the map via the menu.
The size of the land tool still be increased back to what you set it as, but this happens every time you open up the mini-map.

forest frontiers 2018-04-05 17-23-54
And then open up the map...
forest frontiers 2018-04-05 17-23-58

- [Unknown] Reproducible in RCT2 (vanilla)?
- [No] Specific to multiplayer?

**Steps to reproduce:**
1. Open up the land tool
2. Increase the area of its effect to anything larger than 1x1
3. Open up the mini-map

### PR DESCRIPTION
**Pull request by NathanZabriskie**
Fixes #7382: Added local variable for the minimap land rights tool and modified behavior as per @Broxzier's suggestions.

### CODE REVIEW

**Comment by Broxzier** [comment]
Nice, works exactly as expected. :+1:

Please add an entry for this to the [changelog](https://github.com/OpenRCT2/OpenRCT2/blob/develop/distribution/changelog.txt)? And feel free to add yourself to contributors.md file under [Bug fixes](https://github.com/OpenRCT2/OpenRCT2/blob/develop/contributors.md#bug-fixes).

**Comment by NathanZabriskie** [comment]
Thanks! I've updated the files you mentioned. 

Looking forward to working on this project. Just discovered it recently after growing up a huge RCT fan :)

File: distribution/changelog.txt [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - Fix: [#7327] Abstract scenery and stations don't get fully See-Through when hiding them (original bug).
 - Fix: Cut-away view does not draw tile elements that have been moved down on the list.
 - Improved: Raising land near the map edge makes the affected area smaller instead of showing an 'off edge map' error.
+- Fix: [#7382] Opening the mini-map reverts the size of the land tool to 1x1, regardless of what was selected before.
```
Comment (Broxzier) [inline distribution/changelog.txt]: The changelog is ordered by category and then issue number, so this entry should be below issue #7327.

File: contributors.md [hunk 2]
```diff
@@ -106,6 +106,7 @@ The following people are not part of the project team, but have been contributin
 * Justin Gottula (jgottula)
 * Seongsik Park (pss9205)
 * (Deurklink)
+* Nathan Zabriskie
```
Comment (Broxzier) [inline contributors.md]: Please include your username inside parentheses too, even though it's almost the same, for consistency.

**Pull request merged**